src/charts/utilization/utilization-chart.html

```diff
@@ -1,7 +1,7 @@
 <div class="utilization-chart-pf">
-    <span class="title">{{config.title}}</span>
+    <span class="h3">{{config.title}}</span>
```

Comment:
Right, but I feel like we're relying on a bug in patternfly to get the desired styling.  What will happen if they fix `.content .h3` to be the same as `.content h3`.  To me it makes more sense to explicitly override.